pandas/tests/resample/test_resample_api.py

```diff
@@ -771,3 +771,58 @@ def test_end_and_end_day_origin(
     )
 
     tm.assert_series_equal(res, expected)
+
+
+@pytest.mark.parametrize("numeric_only", [True, False])
+def test_downsample_method(numeric_only):
+    # test if `numeric_only` behave as expected for Resampler downsample methods.
```

Comment:
Can you add GH# as a comment on the first line, i.e. `# GH#46442`

Reply:
Thanks @rhshadrach. I will add this tag to my code.